serving/docs/lmi/user_guides/release_notes.md

```diff
@@ -0,0 +1,64 @@
+# LMI V10 containers release
+
+This document will contain the latest releases of our LMI containers. For details on any other previous releases, please refer our [github release page](https://github.com/deepjavalibrary/djl-serving/releases)
+
+## Release Notes
+
+### Release date: June 6, 2024
+
+Check out our latest [Large Model Inference Containers](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#large-model-inference-containers).
```

Comment:
The uri better point to 0.28.0-dlc branch

Reply:
Yes all the uris now point to 0.28.0-dlc branch